forem/forem
# Update following_tags page and create hidden_tags page

## What type of PR is this? (check all applicable)


## Description
This is the an iterative part the of the bigger Hidden Tags/Following Tag functionality. In this PR we create a "hidden tags" page and we update the "following tags" to only show follows with a positive number. The hidden tags page shows following with a negative number to indicate that the tag should be hidden. 

There are a few existing bugs that I point out in the QA section.

## Related Tickets & Documents

- Related Issue #19675 
- Closes #

## QA Instructions, Screenshots, Recordings

1. Follow some tags on /tags
2. You should see them reflect with positive weights on the dashboard/following_tags. 
3. You should not see them reflect on dashboard/hidden_tags.
4. On the dashboard/following_tags page, if you update a weight then it should move to the dashboard/hidden_tags page. 
5. On the dashboard/hidden_tags page, if you update a weight then it should move to the dashboard/following_tags page. 
6. After updating it always redirects to dashboard/following_tags page, I plan on updating it to redirect accordingly in a follow up PR. I think this is okay for now. 
7. Follow as many tags as you can to test that it paginates properly. The limit per page is 80, hence you can rather update the limit via a param dashboard/following_tags?per_page=10 and test it that way. Note: there is an existing bug where we return the first page of data twice, I'll log this separately. Another existing bug is where the newly loaded pagination cards do not have the correct color strip on the card.

https://www.loom.com/share/a20f957ccc5b4376859131705e0c8669?sid=fdc84a78-893f-407c-bf86-8e1e55d42cae

### UI accessibility checklist

N/A

_If your PR includes UI changes, please utilize this checklist:_

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](gif_link)
